### DISCUSSION TITLE
Change default for new tab to powershell instead of cmd

### DISCUSSION THREAD

**Discussion by mrcl-1893** [Eugeny/tabby#9639] [General]
Hi all,

does anybody know how to change the default for a new tab from cmd.exe to powershell.exe?
I would be glad for help ;)

Thank you :)

**Comment by mrcl-1893** [comment]
I found this but there was no solution on the discussion yet:
https://github.com/Eugeny/tabby/discussions/9623

**Comment by Eugeny** [comment]
Settings -> Profiles -> Default profile
